bootstrap/http/src/main/kotlin/com/studentcenter/weave/bootstrap/meeting/dto/MeetingTeamLocationResponse.kt

```diff
@@ -0,0 +1,24 @@
+package com.studentcenter.weave.bootstrap.meeting.dto
+
+import com.studentcenter.weave.domain.meeting.enums.Location
+
+data class MeetingTeamLocationResponse(
+    val locations: List<MeetingTeamLocationDto>
+) {
+
+    data class MeetingTeamLocationDto(
+        val item: String,
+        val value: String
+    )
+
+    companion object {
+
+        fun of(): MeetingTeamLocationResponse {
+            return MeetingTeamLocationResponse(Location.entries.map {
+                MeetingTeamLocationDto(it.name, it.value)
+            })
+        }
```

Comment:
p3) 매번 응답을 만드는 것보다 싱글톤 패턴으로 하나의 인스턴스를 만들고 조회하는 팩토리 메서드를 이용하는게 좋은 것 같아요.

```suggestion
        private lateinit var instance: MeetingTeamLocationResponse

        fun getInstance(): MeetingTeamLocationResponse {
            if (instance == null) {
                instance = MeetingTeamLocationResponse(
                    Location.entries.map {
                        MeetingTeamLocationDto(it.value, it.isCapitalArea)
                    }
                )
            }

            return instance
        }
```